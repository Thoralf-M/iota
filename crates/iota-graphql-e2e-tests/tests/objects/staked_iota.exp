processed 7 tasks

init:
C: object(0,0)

task 1, lines 7-32:
//# run-graphql
Response: {
  "data": {
    "objects": {
      "edges": [
        {
          "cursor": "IL9DP2kbEYuVrsMh/qbvy/UzPbGVC+4DYL62CF/25XvvAAAAAAAAAAA=",
          "node": {
            "asMoveObject": {
              "asStakedIota": {
                "principal": "1500000000000000"
              }
            }
          }
        }
      ]
    },
    "address": {
      "stakedIotas": {
        "edges": []
      }
    }
  }
}

task 2, lines 34-36:
//# programmable --sender C --inputs 10000000000 @C
//> SplitCoins(Gas, [Input(0)]);
//> TransferObjects([Result(0)], Input(1))
created: object(2,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 1960800,  storage_rebate: 0, non_refundable_storage_fee: 0

task 3, line 38:
//# run 0x3::iota_system::request_add_stake --args object(0x5) object(2,0) @validator_0 --sender C
events: Event { package_id: iota_system, transaction_module: Identifier("iota_system"), sender: C, type_: StructTag { address: iota_system, module: Identifier("validator"), name: Identifier("StakingRequestEvent"), type_params: [] }, contents: [51, 172, 177, 34, 21, 213, 71, 92, 252, 12, 178, 2, 74, 150, 207, 59, 239, 86, 133, 54, 89, 150, 189, 251, 124, 97, 39, 140, 166, 241, 176, 243, 175, 163, 158, 79, 0, 218, 226, 120, 249, 119, 199, 198, 147, 10, 94, 44, 118, 232, 93, 23, 165, 38, 215, 36, 187, 206, 15, 184, 31, 176, 125, 76, 140, 202, 78, 28, 224, 186, 89, 4, 206, 166, 29, 249, 36, 45, 162, 247, 210, 158, 62, 243, 40, 251, 126, 192, 124, 8, 107, 59, 244, 124, 166, 26, 0, 0, 0, 0, 0, 0, 0, 0, 0, 228, 11, 84, 2, 0, 0, 0] }
created: object(3,0), object(3,1)
mutated: 0x0000000000000000000000000000000000000000000000000000000000000005, object(0,0)
deleted: object(_), object(2,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 14561600,  storage_rebate: 1960800, non_refundable_storage_fee: 0

task 4, line 40:
//# create-checkpoint
Checkpoint created: 1

task 5, line 42:
//# advance-epoch
Epoch advanced: 0

task 6, lines 44-70:
//# run-graphql
Response: {
  "data": {
    "objects": {
      "edges": [
        {
          "cursor": "IFlKkvFDqa6Fb/O7sSlslqvlkGIX1xGbSJDOVU2K2rgYAgAAAAAAAAA=",
          "node": {
            "asMoveObject": {
              "asStakedIota": {
                "principal": "10000000000",
                "poolId": "0x33acb12215d5475cfc0cb2024a96cf3bef5685365996bdfb7c61278ca6f1b0f3"
              }
            }
          }
        },
        {
          "cursor": "IL9DP2kbEYuVrsMh/qbvy/UzPbGVC+4DYL62CF/25XvvAgAAAAAAAAA=",
          "node": {
            "asMoveObject": {
              "asStakedIota": {
                "principal": "1500000000000000",
                "poolId": "0x33acb12215d5475cfc0cb2024a96cf3bef5685365996bdfb7c61278ca6f1b0f3"
              }
            }
          }
        },
        {
          "cursor": "IPYi3RvwU/w/rb97YP5Cc9ykqR1SkqI2pRfPe6FDWzE1AgAAAAAAAAA=",
          "node": {
            "asMoveObject": {
              "asStakedIota": {
                "principal": "15340000000000",
                "poolId": "0x33acb12215d5475cfc0cb2024a96cf3bef5685365996bdfb7c61278ca6f1b0f3"
              }
            }
          }
        }
      ]
    },
    "address": {
      "stakedIotas": {
        "edges": [
          {
            "cursor": "IFlKkvFDqa6Fb/O7sSlslqvlkGIX1xGbSJDOVU2K2rgYAgAAAAAAAAA=",
            "node": {
              "principal": "10000000000"
            }
          }
        ]
      }
    }
  }
}
