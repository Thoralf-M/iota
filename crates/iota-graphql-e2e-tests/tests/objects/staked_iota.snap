---
source: external-crates/move/crates/move-transactional-test-runner/src/framework.rs
---
processed 7 tasks

init:
C: object(0,0)

task 1, lines 7-32:
//# run-graphql
Response: {
  "data": {
    "objects": {
      "edges": [
        {
          "cursor": "IIW0jOcDJk6WkCjTAtEAIUXzIxZhmFDZ5QYn63b2I9crAAAAAAAAAAA=",
          "node": {
            "asMoveObject": {
              "asStakedIota": {
                "principal": "1500000000000000"
              }
            }
          }
        }
      ]
    },
    "address": {
      "stakedIotas": {
        "edges": []
      }
    }
  }
}

task 2, lines 34-36:
//# programmable --sender C --inputs 10000000000 @C
//> SplitCoins(Gas, [Input(0)]);
//> TransferObjects([Result(0)], Input(1))
created: object(2,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 1960800,  storage_rebate: 0, non_refundable_storage_fee: 0

task 3, line 38:
//# run 0x3::iota_system::request_add_stake --args object(0x5) object(2,0) @validator_0 --sender C
events: Event { package_id: iota_system, transaction_module: Identifier("iota_system"), sender: C, type_: StructTag { address: iota_system, module: Identifier("validator"), name: Identifier("StakingRequestEvent"), type_params: [] }, contents: [134, 1, 154, 68, 53, 144, 148, 137, 199, 71, 208, 26, 192, 248, 149, 59, 208, 212, 27, 244, 119, 108, 221, 182, 171, 144, 19, 240, 54, 198, 164, 135, 175, 163, 158, 79, 0, 218, 226, 120, 249, 119, 199, 198, 147, 10, 94, 44, 118, 232, 93, 23, 165, 38, 215, 36, 187, 206, 15, 184, 31, 176, 125, 76, 140, 202, 78, 28, 224, 186, 89, 4, 206, 166, 29, 249, 36, 45, 162, 247, 210, 158, 62, 243, 40, 251, 126, 192, 124, 8, 107, 59, 244, 124, 166, 26, 0, 0, 0, 0, 0, 0, 0, 0, 0, 228, 11, 84, 2, 0, 0, 0] }
created: object(3,0), object(3,1)
mutated: 0x0000000000000000000000000000000000000000000000000000000000000005, object(0,0)
deleted: object(_), object(2,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 14630000,  storage_rebate: 1960800, non_refundable_storage_fee: 0

task 4, line 40:
//# create-checkpoint
Checkpoint created: 1

task 5, line 42:
//# advance-epoch
Epoch advanced: 0

task 6, lines 44-70:
//# run-graphql
Response: {
  "data": {
    "objects": {
      "edges": [
        {
          "cursor": "IAjJcAqfhjsX0MHoNN48qciDbuSE5M7YqsSOeHCHnktqAgAAAAAAAAA=",
          "node": {
            "asMoveObject": {
              "asStakedIota": {
                "principal": "10000000000",
                "poolId": "0x86019a4435909489c747d01ac0f8953bd0d41bf4776cddb6ab9013f036c6a487"
              }
            }
          }
        },
        {
          "cursor": "IEKlC1S9ir3zTui2XEFGU8gnBYggv5MiqmufAsOnD2XyAgAAAAAAAAA=",
          "node": {
            "asMoveObject": {
              "asStakedIota": {
                "principal": "15340000000000",
                "poolId": "0x86019a4435909489c747d01ac0f8953bd0d41bf4776cddb6ab9013f036c6a487"
              }
            }
          }
        },
        {
          "cursor": "IIW0jOcDJk6WkCjTAtEAIUXzIxZhmFDZ5QYn63b2I9crAgAAAAAAAAA=",
          "node": {
            "asMoveObject": {
              "asStakedIota": {
                "principal": "1500000000000000",
                "poolId": "0x86019a4435909489c747d01ac0f8953bd0d41bf4776cddb6ab9013f036c6a487"
              }
            }
          }
        }
      ]
    },
    "address": {
      "stakedIotas": {
        "edges": [
          {
            "cursor": "IAjJcAqfhjsX0MHoNN48qciDbuSE5M7YqsSOeHCHnktqAgAAAAAAAAA=",
            "node": {
              "principal": "10000000000"
            }
          }
        ]
      }
    }
  }
}
